sections/docker.zsh

```diff
@@ -65,6 +74,10 @@ spaceship_docker() {
     docker_host=" via ("$(basename $DOCKER_HOST | cut -d':' -f1)")"
   fi
 
+  if [[ -n $docker_context ]]; then
+    docker_host=" via ($docker_context)"
```

Comment:
I don't really like the way we did that. I think it has to be done like we did with the `git` section and it's subsections.

Reply:
Sure, I can refactor it into two sections if you want.